common/config/rush/pnpm-lock.yaml

```diff
@@ -12410,22 +12410,23 @@ packages:
       '@azure/identity': 4.4.1
       '@microsoft/api-extractor': 7.47.9(@types/node@18.19.54)
       '@types/node': 18.19.54
-      '@vitest/browser': 2.1.2(playwright@1.47.2)(typescript@5.6.2)(vitest@2.1.2)
+      '@vitest/browser': 2.1.2(playwright@1.47.2)(typescript@5.5.4)(vitest@2.1.2)
       '@vitest/coverage-istanbul': 2.1.2(vitest@2.1.2)
       dotenv: 16.4.5
-      eslint: 9.12.0
+      eslint: 8.57.1
```

Comment:
downgrading the version here ?

Reply:
Updated package.json and here has no differences anymore.